Whiskey/Tasks/Parallel.ps1

```diff
@@ -2,6 +2,10 @@ function Invoke-WhiskeyParallelTask
 {
     [CmdletBinding()]
     [Whiskey.Task('Parallel')]
+    [Whiskey.RequiresPowerShellModule('ThreadJob',

```

Comment:
Where and how does this module get used? I don't see any other reference to it in the task.

Reply:
Whoops. I removed usages of the `ThreadJob` module but didn't remove the `RequiresPowerShellModule` attributes. I tried it out to replace Start-Job and didn't work.

Reply:
Ok there's references to `ThreadJob` in multiple other tasks you'll want to remove as well I think.